plugins/inlinetoolbar/skins/moono/inlinetoolbar.css

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or http://ckeditor.com/license
+*/
+
+.cke_balloon.cke_inlinetoolbar
+{
+	background: linear-gradient(to bottom, #f5f5f5, #cfd1cf);
```

Comment:
I know ATM we are not targeting older browsers probably, but it is good to keep this in mind. So the rule should be same as for standard toolbar (including `filter` for IEs):

![image](https://user-images.githubusercontent.com/1061942/31495015-188c49a0-af56-11e7-9e26-77784bf83e94.png)